csharp/ql/src/Security Features/CWE-601/examples/UrlRedirectGood.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Web;
+using System.Collections.Generic;
+
+public class UnvalidatedUrlHandler : IHttpHandler
+{
+    private List<string> VALID_REDIRECTS = new List<string>{ "http://cwe.mitre.org/data/definitions/601.html", "http://cwe.mitre.org/data/definitions/79.html" };
```

Comment:
nit: Use the brand new C# 12 collection expression language feature
```suggestion
    private List<string> VALID_REDIRECTS = ["http://cwe.mitre.org/data/definitions/601.html", "http://cwe.mitre.org/data/definitions/79.html"];
```

Reply:
I prefer not to use language features that are that new.